### ISSUE TITLE
Plugin needs to clean up after itself

### ISSUE THREAD

**Issue by senlin** [byepass-co/byepass-wordpress#1]
Either upon deactivation, but surely after removal of the plugin, it needs to remove the 3 entries from the database, especially since on the settings screen nothing can be changed after initial configuration.

**Comment by dommholland** [comment]
Agreed m.  Should implement removal on delete/deactivation and also from settings page manually.

**Comment by dommholland** [comment]
Byepass is 3rd party and external to either site so it creates a cross-site sessions.  Makes logging in easier when you use multiple websites that all utilise Byepass.  The more sites that use Byepass the easier it is to login around the web!

**Comment by senlin** [comment]
I think this comment is meant for #2?
